ui/src/components/context-menu/context-menu.tsx

```diff
@@ -125,6 +125,7 @@ export const ContextMenu = ({
               }}
               onClick={(e) => {
                 if (!item.disabled && item.onClick) {
+                  e.stopPropagation();
```

Comment:
This prevents the onclick event from bubbling up when the context menu should actually be handling the event